configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionCache.java

```diff
@@ -1,17 +1,17 @@
-// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.config.server.session;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 
 /**
- * A generic session repository that can store any type of session that extends the abstract interface.
+ * A session cache that can store any type of session that extends the abstract interface.
```

Comment:
We're probably talking about the Session class here, so I suggest to add a javadoc link to the class instead of the slightly confusing 'session that extends the abstract interface'.

Reply:
Fixed